analytics/analytics.go

```diff
@@ -5,91 +5,121 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"os"
 	"time"
 
-	log "github.com/Sirupsen/logrus"
-	"github.com/bitrise-core/bitrise-plugins-analytics/configs"
 	models "github.com/bitrise-io/bitrise/models"
+	"github.com/bitrise-io/bitrise/plugins"
+	"github.com/lunny/log"
```

Comment:
That's not our log package. It should be `"github.com/bitrise-io/go-utils/log"` instead.

Reply:
uhh, very good catch